e2e/helpers/generated_order_helper.js

```diff
@@ -168,11 +168,13 @@ module.exports = {
     }
   },
 
-  async assertOrder(I, caseViewPage, order, defaultIssuedDate, hasAllocatedJudge = false) {
+  async assertOrder(I, caseViewPage, order, defaultIssuedDate, hasAllocatedJudge = false, isOrderRemoved = false) {
     caseViewPage.selectTab(caseViewPage.tabs.orders);
     const numberOfOrders = await I.grabNumberOfVisibleElements('//*[text() = \'Type of order\']');
-    const orderHeading = `Order ${numberOfOrders}`;
-    I.seeInTab([orderHeading, 'Type of order'], order.fullType);
+    let orderHeading = `Order ${numberOfOrders}`;
+
+    isOrderRemoved ?
+      orderHeading = `Removed orders ${numberOfOrders}` : orderHeading = `Order ${numberOfOrders}`;
```

Comment:
Cannot this just be single line with const?

Reply:
you mean for orderHeading? I don't think so, because i am changing orderHearing based on isOrderRemoved.

Reply:
Have changed to
```js
const orderHeading = isOrderRemoved ? `Removed orders ${numberOfOrders}` : `Order ${numberOfOrders}`;
```

Reply:
ahh ok :), i misunderstood that.